home.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Nick Clifford</title>
+    <link rel="stylesheet" href="homestyle.css">
+	<meta charset="utf-8">
```

Comment:
Same for this file, the charset meta tag is duplicated and there's no responsive meta tag